packages/ember-metal/lib/computed.js

```diff
@@ -127,9 +127,10 @@ const DEEP_EACH_REGEX = /\.@each\.[^.]+\./;
   @namespace Ember
   @public
 */
-function ComputedProperty(config, opts) {
+function ComputedProperty(config, opts = {}) {
```

Comment:
this will compile to something that touches arguments.length. just initialize this._dependentKeys and this._readOnly with a `opts === undefined`

The miss for {} is a waste too.